f2/src/locales/en.json

```diff
@@ -26,20 +41,14 @@
   "confirmationPage.title": "Review your report",
   "contactinfoPage.backButton": "Back to the impact",
   "contactinfoPage.emailAddress": "Email address",
+  "contactinfoPage.emailAddressExample": "Format: letters or letter plus number@letters.letters, For example: adfg1@gmail.com",
   "contactinfoPage.fullName": "Full name",
+  "contactinfoPage.fullNameExample": "Format: FirstName space LastName, For example: George Smith",
   "contactinfoPage.intro": "We will use this to send you a confirmation email and to link your report to other reports in your area.",
   "contactinfoPage.nextButton": "Next: Confirm information",
   "contactinfoPage.postCode": "Postal code",
+  "contactinfoPage.postCodeExample": "For example: K3H5J6",
   "contactinfoPage.title": "Leave your contact information",
-  "fileUpload.added": "File added",
-  "fileUpload.removed": "File removed",
-  "impactPage.affected1": "Device or account hacked",
-  "impactPage.affected2": "Information stolen",
-  "impactPage.affected3": "Money lost",
-  "impactPage.affected4": "Reputation damaged",
-  "impactPage.affected5": "Safety threatened",
-  "impactPage.affected6": "Wellbeing affected",
-  "impactPage.affected7": "No impact",
```

Comment:
`npm run extract` deleted all these lines (and added others) when it really shouldn't have. For now we should change `en.json` and `fr.json` here to only change the `contactinfoPage...` lines. PR #985 will fix `npm run extract`'s behaviour.